configure

```diff
@@ -4112,6 +4116,36 @@ fi
 
 fi
 
+# REPORTS_BASE_URL definition
+
+
+
+# Check whether --with-reports-base-url was given.
+if test "${with_reports_base_url+set}" = set; then :
+  withval=$with_reports_base_url;
+  case $withval in
+    yes)
+      as_fn_error $? "argument required for --with-reports-base-url option" "$LINENO" 5
+      ;;
+    no)
+      as_fn_error $? "argument required for --with-reports-base-url option" "$LINENO" 5
+      ;;
+    *)
+      REPORTS_BASE_URL="$withval"
```

Comment:
(if we do `with_reports_hostname` we'd change this to something like `"https://${withval}"`

Reply:
Done.